Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -305,6 +305,12 @@
           "Name": "DoNotExposePlaintextSecrets/ConvertToSecureString",
           "ContentsRegex": "$SEC101/052.ConvertToSecureString",
           "MessageArguments": { "secretKind": "ConvertTo-SecureString powershell" }
+        },
+        {
+          "Id": "SEC101/054",
+          "Name": "DoNotExposePlaintextSecrets/AlibabaAccessKey",
+          "ContentsRegex": "$SEC101/054.AlibabaAccessKey",
+          "MessageArguments": { "secretKind": "Alibaba Access Key" }
```

Comment:
>Alibaba Access Key [](start = 47, length = 18)

s/be cased 'Alibaba access key'.

Reply:
Done

---
In reply to: [591876969](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591876969) [](ancestors = 591876969)